tests/test_schema.py

```diff
@@ -124,15 +124,41 @@ def test_invalid_amino(self):
 
 class TestEdgeSchema(unittest.TestCase):
     """Tests for the jsonschema edge validation."""
-
-    def test_predefined_example(self):
-        """Test a predefined edge example."""
+    
+    @staticmethod
+    def _load_example() -> dict:
```

Comment:
when you're writing unit tests, you can actually save some state between all of the tests by using the class method`TestCase.setUpClass()`

```python
@classmethod
def setUpClass(cls):
    self.edge = ...
```

Then `self.edge` is available throughout all of the tests in this class (don't modify it though, because the change will persist between test functions)